server/repository/src/migrations/v1_02_01/mod.rs

```diff
@@ -0,0 +1,33 @@
+use super::{version::Version, Migration};
+
+use crate::StorageConnection;
+pub(crate) struct V1_02_01;
+
+impl Migration for V1_02_01 {
```

Comment:
the migrations need to be added to `/server/repository/src/migrations/mod.rs` too 🙏

Reply:
[20c0cd1](https://github.com/openmsupply/open-msupply/pull/2038/commits/20c0cd1c18240acbf8cd92b698cad41de916ffc2)